### PR TITLE
3 packages from c-cube/printbox at 0.6

### PR DESCRIPTION
**Pull request by c-cube**
This pull-request concerns:
-`printbox.0.6`: Allows to print nested boxes, lists, arrays, tables in several formats
-`printbox-html.0.6`: Printbox unicode handling
-`printbox-text.0.6`: Text renderer for printbox, using unicode edges

---
* Homepage: https://github.com/c-cube/printbox/
* Source repo: git+https://github.com/c-cube/printbox.git
* Bug tracker: https://github.com/c-cube/printbox/issues/

---
:camel: Pull-request generated by opam-publish v2.1.0

### CODE REVIEW

**Comment by kit-ty-kate** [comment]
Same problem with your setup-ocaml CI setup as in https://github.com/ocaml/opam-repository/pull/20345

File: packages/printbox-text/printbox-text.0.6/opam
Comment (kit-ty-kate) [inline packages/printbox-text/printbox-text.0.6/opam:11]:
```suggestion
  "dune" { >= "2.0" }
```

**Comment by kit-ty-kate** [comment]
Could you add the missing license fields to those packages as well?

File: packages/printbox-html/printbox-html.0.6/opam
Comment (kit-ty-kate) [inline packages/printbox-html/printbox-html.0.6/opam:17]: you removed my `"odoc" {with-doc}` addition, which would make the documentation fail without it (doc dependencies are not transitive, same as the test dependencies)

Comment (c-cube) [reply packages/printbox-html/printbox-html.0.6/opam:17]: I removed it because it wasn't added upstream and opam-publish doesn't automatically fetch from this PR. I'll add it back.

**Review by kit-ty-kate — APPROVED** [review]
LGTM once CI is green

File: packages/printbox-text/printbox-text.0.6/opam
Comment (mseri) [inline packages/printbox-text/printbox-text.0.6/opam:17]:
```suggestion
  "uucp" { >= "1.1.0" }
```
Uses `Uucp.Break.tty_width_hint`

File: packages/printbox-html/printbox-html.0.6/opam
Comment (mseri) [inline packages/printbox-html/printbox-html.0.6/opam:18]:
```suggestion
  "tyxml" {>="4.0"}
```
This is where the `Tyxml` module was introduced

**Comment by mseri** [comment]
I am going to wait until the upper bounds are merged before applying the fixes here and re-running the CI

File: packages/printbox-text/printbox-text.0.6/opam
Comment (mseri) [inline packages/printbox-text/printbox-text.0.6/opam:17]:
```suggestion
  "uucp" { >= "2.0" }
```
Turns out you need 2.0 since you are using `Uchar.t`

Comment (mseri) [reply packages/printbox-text/printbox-text.0.6/opam:17]:
```
- File "src/printbox-text/PrintBox_text.ml", line 78, characters 60-61:
- Error: This expression has type Uchar.t
-        but an expression was expected of type Uucp.uchar = int
```

**Comment by mseri** [comment]
@c-cube can you upstream the opam file changes?

**Comment by c-cube** [comment]
Thank you!

**Pull request merged**
